### PR TITLE
Follow official Node.js Docker image best practices

### PR DESCRIPTION
**Pull request by giorgiosironi**
From https://github.com/nodejs/docker-node/blob/master/docs/BestPractices.md#non-root-user:
 
- [x] Run as `node` user prevents
  - privilege escalation to the host in case of Docker vulnerability
  - write access to all of the container's files from JavaScript code
- [x] Document that `npm run` is acting as PID 1, handling signals to stop the application

### CODE REVIEW

**Comment by giorgiosironi** [comment]
Added https://github.com/orgs/libero/teams/infrastructure-reviewers with `Read` permissions for review requests.

File: Dockerfile [hunk 1]
```diff
@@ -76,4 +76,7 @@ ENV NODE_ENV=production
 COPY --from=npm-prod /app/ .
 COPY --from=build-prod /app/build/ build/
 
+USER node
```
Comment (thewilkybarkid) [inline Dockerfile:79]: Guess we do want this after things have been copied in?

Comment (thewilkybarkid) [reply Dockerfile:79]: And no need to replicate on `dev`.

Comment (thewilkybarkid) [reply Dockerfile:79]: Just read the guidance, makes sense to only include it at the end of `prod`.

**Review by thewilkybarkid — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (giorgiosironi) [reply Dockerfile:79]: `COPY` doesn't follow `USER` for file ownership, but `RUN npm ...` will; so my default is to switch at the end as I'm not very concerned about `root` in the build phase. Can be moved up to just after `npm run build` but not very useful to do so.

**Pull request merged**